lib/charted.js

```diff
@@ -9,16 +9,19 @@ module.exports = function(app, charted_root) {
   if (charted_root === undefined) { charted_root = '/charted/' }
 
   app.use(charted_root + 'get', function (req, res, next) {
-    var get_url = url.parse(req.url, true).query.url
```

Comment:
What was the point for removing this variable if you ended up using `urlObj.query.url` everywhere else?